code/game/objects/objs.dm

```diff
@@ -461,3 +461,7 @@
 /// override for subtypes that require extra behaviour when spawned from a vendor
 /obj/proc/post_vendor_spawn_hook(mob/living/carbon/human/user)
 	return
+
+/// Overriden for subtypes of loadout items that require extra behavior
+/obj/proc/post_loadout_spawn(mob/living/carbon/human/user)
+	return
```

Comment:
That right here is half the reason we have signals - don't define an /obj proc for a niche handler on a couple items